### PR TITLE
Add Moby TSC references/governance details

### PR DESCRIPTION
**Pull request by estesp**
Also added back some of the maintainer processes that were in
MAINTAINERS but moved to docker/opensource repo. I believe this
project's governance should be disconnected from docker/opensource as
project's remaining under docker/opensource will not use the Moby TSC.

Docker-DCO-1.1-Signed-off-by: Phil Estes <estesp@linux.vnet.ibm.com>

### CODE REVIEW

File: MAINTAINERS
Comment (thaJeztah) [inline MAINTAINERS]: Was discussing this with @AkihiroSuda at some point; wondering if we should have a `moby/projects` (e.g.) repository that allows collecting the maintainers of all repos, and where we can share common documents

**Review by thaJeztah — APPROVED** [review]
LGTM

ping @moby/moby-maintainers

**Comment by thaJeztah** [comment]
Wow, you're still using the `Docker-DCO-1.1-Signed-off-by` sign off 😄

**Comment by estesp** [comment]
> Wow, you're still using the Docker-DCO-1.1-Signed-off-by sign off 😄

How else are we going to continuously CI-validate @tianon's scripts for parsing sign-off?! In other news, that's buried in some git commit hook script I put in my `.git` over 3 years ago and have long forgotten about :)

**Comment by tianon** [comment]
@estesp I was honestly about to comment something about PTSD as soon as I saw @thaJeztah's comment :laughing:

Spent wayyy too much time on validating all that.

This PR seems like a good approach to me. :+1:

File: CONTRIBUTING.md
Comment (mistyhacks) [inline CONTRIBUTING.md]: "this Moby project"? As opposed to all the other ones?

Also you need to introduce the acronym TSC somewhere (wouldn't want people to think you switched to talking about [Tractor Supply Company](https://www.tractorsupply.com/) so why not do it in the link text?

**Review by mdlinville — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md
Comment (thaJeztah) [reply CONTRIBUTING.md]: 🚜

File: MAINTAINERS
Comment (AkihiroSuda) [reply MAINTAINERS]: Probably we can use TSC repo?

Comment (thaJeztah) [reply MAINTAINERS]: I'm okay with that. Currently the repository contains the "rules" as well (perhaps we should remove that, because repository can keep separate rules), the repo is primarily used as an easy way to have contact information for all maintainers (so currently both "docker" open source and "moby"), not sure if that's a problem; if we should split, or not

**Review by vdemeester — APPROVED** [review]
LGTM 👍

**Review by coolljt0725 — APPROVED** [review]
:+1:  LGTM

File: CONTRIBUTING.md
Comment (estesp) [reply CONTRIBUTING.md]: Thanks @mistyhacks for the review! I believe I dealt with both comments; PTAL! (The weird construct with "this Moby project" was an attempt to reference that many Moby projects (7 of them?) are under the TSC governance, not just the moby/moby repo, but definitely didn't read well anyway) :)

**Review by yongtang — APPROVED** [review]
LGTM. Thanks! 👍

File: project/GOVERNANCE.md
Comment (mistyhacks) [inline project/GOVERNANCE.md]: s/will contact/contacts

"invite them to open"

File: project/GOVERNANCE.md
Comment (mistyhacks) [inline project/GOVERNANCE.md]: Maybe "Either at their own request or due to project inactivity." Maybe link to the inactivity section below.

File: project/GOVERNANCE.md
Comment (mistyhacks) [inline project/GOVERNANCE.md]: Maybe "How to step down"

File: project/GOVERNANCE.md
Comment (mistyhacks) [inline project/GOVERNANCE.md]: I think you can just remove the first clause. Maybe "An existing maintainer" and then continue in the singular for the rest of the section. I assume this is not usually an activity that happens to multiple people at once.

File: project/GOVERNANCE.md
Comment (mistyhacks) [inline project/GOVERNANCE.md]: What is a neutral person?

File: project/GOVERNANCE.md
Comment (mistyhacks) [inline project/GOVERNANCE.md]: Maybe "continue in the role"

File: project/GOVERNANCE.md
Comment (mistyhacks) [inline project/GOVERNANCE.md]: nit: comma after duties

Is it really 60% to add but 66% to remove?

File: project/GOVERNANCE.md
Comment (mistyhacks) [inline project/GOVERNANCE.md]: Maybe something about "no ad hominem attacks" here. I could see this kind of discussion getting ugly.

Clarify whether the person is eligible to vote on their own removal?

File: project/GOVERNANCE.md
Comment (mistyhacks) [inline project/GOVERNANCE.md]: I don't think you need to shout this. :)

File: project/GOVERNANCE.md
Comment (mistyhacks) [inline project/GOVERNANCE.md]: s/This means that//

Instead of caps, why not use **bold**?

File: project/GOVERNANCE.md
Comment (mistyhacks) [inline project/GOVERNANCE.md]: Nit, need period after APIs.

File: project/GOVERNANCE.md
Comment (mistyhacks) [inline project/GOVERNANCE.md]: s/all decisions/each decision (and correct for agreement following)

File: project/GOVERNANCE.md
Comment (mistyhacks) [inline project/GOVERNANCE.md]: Maybe s/is/is expressed as

File: project/GOVERNANCE.md
Comment (mistyhacks) [inline project/GOVERNANCE.md]: the moby/moby repository, perhaps

File: project/GOVERNANCE.md
Comment (mistyhacks) [inline project/GOVERNANCE.md]: Great place to use a numbered list. Maybe don't state that it is 3 steps because there may be more later and then you'd have to remember to update that number too. ;)

File: project/GOVERNANCE.md
Comment (mistyhacks) [inline project/GOVERNANCE.md]: s/refuse/close or reject

File: project/GOVERNANCE.md
Comment (mistyhacks) [inline project/GOVERNANCE.md]: Pull requests are reviewed by the current...

File: project/GOVERNANCE.md
Comment (mistyhacks) [inline project/GOVERNANCE.md]: are organized to synchronously discuss tricky PRs, as well as design and architecture decisions.

File: project/GOVERNANCE.md
Comment (mistyhacks) [inline project/GOVERNANCE.md]: s/via/among

File: project/GOVERNANCE.md
Comment (mistyhacks) [inline project/GOVERNANCE.md]: s/via/by

**Review by mdlinville — CHANGES_REQUESTED** [review]
OK, there was a lot more text added, so I reviewed it! Sorry if this is a lot of comments. Most of them are tiny.

**Comment by estesp** [comment]
@mistyhacks thanks for the review. I definitely think it's reasonable to improve the text, and I'll go through your comments and apply a majority of them. The background is that after the split of "Docker" and "Moby" there hasn't been a clear place for the overall project governance/details that have moved to `docker/opensource`. What I did in this PR is made a verbatim copy of the "TOML" format file from `docker/opensource` into `project/GOVERNANCE.md` in this repo (which had misleading/outdated information anyway), and added some markdown section headers, but left the rest of the text intact. Your edits are to that text, and as you noted, use of actual markdown features are missing given it was a "TOML" file, not a markdown file.

Anyway, again, reasonable to improve the text now, although someone should decide if those edits should go back to `docker/opensource` TOML version, but that is outside the scope of my PR for now :)

**Comment by thaJeztah** [comment]
I can update docker/opensource at some point and take those improvement there

File: MAINTAINERS
Comment (estesp) [reply MAINTAINERS]: This is a TOML file; there is no way to specify links, and it does not generate a readable/rich text format, so links wouldn't do anything

File: MAINTAINERS
Comment (estesp) [reply MAINTAINERS]: this is a TOML file; there is no post-processing or generation to any other rich format. Comments in the file are any line prefixed with `#` so I think this is OK.

File: project/GOVERNANCE.md
Comment (estesp) [reply project/GOVERNANCE.md]: no public access to maintainers ML so don't have a link that can be used here

File: project/GOVERNANCE.md
Comment (estesp) [reply project/GOVERNANCE.md]: Removed anything about repository as this text is now being used only for `moby/moby`

File: project/GOVERNANCE.md
Comment (estesp) [reply project/GOVERNANCE.md]: No idea.. changing it to not express anything about the person doing the notification

File: project/GOVERNANCE.md
Comment (estesp) [reply project/GOVERNANCE.md]: no, both say 66%

**Comment by estesp** [comment]
Phew. Unless I made a comment to a comment, all the other changes are applied and branch re-pushed. PTAL :)

**Review by mdlinville — APPROVED** [review]

**Review by thaJeztah — APPROVED** [review]
still LGTM 👍

**Review by mlaventure — APPROVED** [review]
LGTM

**Comment by vdemeester** [comment]
Let's get this in 🎉

**Pull request merged**
